src/cljdoc/render/articles.clj

```diff
@@ -11,6 +11,18 @@
        (assoc version-entity :article-slug)
        (routes/url-for :artifact/doc :path-params)))
 
+(defn prev-next-navigation [{:keys [prev-page next-page version-entity]}]
+  [:div.flex.justify-between
+   (if prev-page
```

Comment:
Should this be a when as well?

Reply:
No, because if `prev-page` is `nil` we want to render empty`div` element, so `next-page` link always will be on the right side